docs/source/deploying/performance.md

```diff
@@ -1,9 +1,9 @@
 ---
 html_meta:
-  "description": "How to improve initial page load performance of above-the-fold content with critical.css."
-  "property=og:description": "How to improve initial page load performance of above-the-fold content with critical.css."
-  "property=og:title": "critical.css (above the fold) optimizations"
-  "keywords": "Volto, Plone, frontend, React, critical.css, optimizations"
+  'description': 'How to improve performance of applications built around Volto stack'
+  'property=og:description': 'How to improve performance of applications built around Volto stack'
+  'property=og:title': 'Performance'
+  'keywords': 'Volto, Plone, frontend, React, optimizations'
```

Comment:
Please do not change `"` to `'`. Revert.

If you change the title here, you also must change the title below on line 9.

The description and keywords should be align with the entire page. I don't think your changes do that.

Overall, I don't see an improvement with the above changes.

Reply:
@stevepiercy From what I see is, the page will be intended for all types of performance improvements, "not just critical css"!

Reply:
> Please do not change " to '. Revert.

Ah, my editor. Again...